tools/translations/ru.po

```diff
@@ -734,7 +734,7 @@ msgstr "Нет совпадений"
 
 #: tools/editor/code_editor.cpp
 msgid "Replaced %d Ocurrence(s)."
-msgstr "Заменено %d совпадений."
+msgstr "Заменено совпадений: %d."
```

Comment:
OK, I'll let you guys discuss with each other what you find best as translations. I can't review myself so I'll just merge what gets PR'ed :P